app/views/articles/show.html.erb

```diff
@@ -31,6 +31,53 @@
     </script>
   <% end %>
 
+<script type="application/ld+json">
```

Comment:
I tested this after fixing the commas with https://search.google.com/structured-data/testing-tool/u/0/ and it flagged a warning which seems reasonable to me: `The dateModified field is recommended. Please provide a value if available.`. 

Example:

```json
{
    "@context": "http://schema.org",
    "@type": "Article",
    "mainEntityOfPage": {
      "@type": "WebPage",
      "@id": "https://dev.to/http://localhost:3000/funk_damon/nine-coaches-waiting-qui-explicabo-2050"
    },
    "url": "http://localhost:3000/funk_damon/nine-coaches-waiting-qui-explicabo-2050",
    "image": "https://res.cloudinary.com/Optional/image/fetch/s--tRcg02ro--/c_imagga_scale,f_auto,fl_progressive,h_500,q_auto,w_1000/https://pigment.github.io/fake-logos/logos/medium/color/2.png",
    "publisher": {
      "@context": "http://schema.org",
      "@type": "Organization",
      "name": "DEV(local) Community",
      "logo": {
        "@context": "http://schema.org",
        "@type": "ImageObject",
        "url": "https://res.cloudinary.com/Optional/image/fetch/c_scale,fl_progressive,q_auto,w_192/f_auto/https://practicaldev-herokuapp-com.freetls.fastly.net/assets/devlogo-pwa-512.png",
        "width": "192",
        "height": "192"
      }
    },
    "headline": " Nine Coaches Waiting Qui explicabo",
    "author": {
      "@context": "http://schema.org",
      "@type": "Person",
      "url": "http://localhost:3000/funk_damon",
      "name": "Damon Funk",
    },
    "datePublished": "2020-04-01T12:16:23Z"
  }
```

Reply:
I'll make this change! I think I had `dateModified` here previously and removed it. 🤔